CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Check on tarantool box initialization before init (now it returns a nice error).
+- Possibility to pass port to `memcached.create()` as a number (#108).
+
+### Changed
+
+- Documentation has been updated.
+- Update an upper value for `readahead` parameter.
+- Switched to using the module's own slab arena. For information `memcached.slab.info()`
+  can be used.
+
+### Fixed
```

Comment:
Fixed #80, fixed #56 -- it is not mentioned. Finished #59 in PR #111 -- no info. #96 is mentioned, but without the issue number. Bundling of small and msgpuck into packages is not mentioned.

Reply:
A build fail is visible, when you're installing the module using luarocks/tarantoolctl rocks. Segfault is definitely visible. Libs bundling affects package content and dependencies -- seems quite visible too.

Reply:
Added information about segfault and bundled lidraries.
The problem with libmemcached is still relevant (#115).

Reply:
If you want to add some points, write and I will add them. But I sincerely think it is better not to say that we fixed the tests with libmemcached because it does not work on popular distributions such as Arch or Ubuntu 22.04.